src/style-stars--4.css

```diff
@@ -0,0 +1,3 @@
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
```

Comment:
The path to the background image in the `url` function should be checked for correctness. If the `images` directory is not at the same level as the `style-stars--4.css` file, the relative path may be incorrect. Ensure the path accurately points to the location of the `star-active.svg` file within the project structure.